sdk/servicebus/service-bus/test/stress/stressTestsBase.ts

```diff
@@ -33,21 +33,39 @@ appInsights
 
 export const defaultClient = appInsights.defaultClient;
```

Comment:
```suggestion
export const appInsightsClient = appInsights.defaultClient;
```
?

Reply:
Let's leave it for now. I was thinking of just removing it altogether (defaultclient is basically a singleton) but I didn't get around to it in this PR.